### PR TITLE
Cenario tesoura X pedra

### PR DESCRIPTION
**Pull request by eminetto**
Connected to #8

### CODE REVIEW

File: src/JokenPo/JokenPo.php [hunk 1]
```diff
@@ -16,4 +16,9 @@ public function rules()
             - Tesoura empata com Tesoura e ganha de Papel
             - Papel empata com Papel e ganha de Pedra';
     }
+
+    public function TesouraXPedra()
```
Comment (eminetto) [inline src/JokenPo/JokenPo.php]: The method TesouraXPedra is not named in camelCase.

**Comment by eminetto** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 1 possible new issue (including those that may have been commented here).

You can see more details about this review at https://ebertapp.io/github/Coderockr/exercicio-git/pulls/21.

**Comment by eminetto** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 1 fixed issue! 🎉

You can see more details about this review at https://ebertapp.io/github/Coderockr/exercicio-git/pulls/21.

File: tests/JokenPoTest.php [hunk 2]
```diff
@@ -10,6 +10,12 @@ public function testPapelXPapel()
         $this->assertEquals('empate', $jokenpo->run('papel', 'papel'));
     }
 
+    public function testTesouraXPedra()
+    {
+        $jokenpo = new JokenPo;
+        $this->assertEquals('pedra', $jokenpo->run('tesoura', 'pedra'));
```
Comment (jeanCarloMachado) [inline tests/JokenPoTest.php]: Dá pra usar static no assert que pula uma função ;P.

**Pull request merged**